src/trace/span-inferrer.ts

```diff
@@ -52,6 +55,10 @@ export class SpanInferrer {
     return "sync";
   }
 
+  isTracedAuthorizerInvocation(event: any): boolean {
+    return event.requestContext?.authorizer?._datadog;
```

Comment:
Is there any danger from not explicitly coalescing this into a bool?

Reply:
It could evaluate true if the key `_datadog` is present but empty, or an empty string or something - but the next call is wrapped in a try block anyway (to handle the said case), so I think it's okay for the usage as is - but I'll add an explicit check for `!== undefined`, in case someone re-uses it in the future.